draft-ietf-quic-transport.md

```diff
@@ -2615,8 +2615,10 @@ Any frame type that mentions a stream ID can be sent in this state.
 
 A stream that is in the "half-closed (local)" state MUST NOT be used for sending
 on new STREAM frames.  Retransmission of data that has already been sent on
-STREAM frames is permitted.  An endpoint MAY also send MAX_STREAM_DATA and
-STOP_SENDING in this state.
+STREAM frames is permitted.  If the stream is canceled or reset while in this
+state, a RST_STREAM frame MAY be sent. A RST_STREAM frame sent in this state
```

Comment:
Are cancelled or reset different?

Reply:
It's probably useful to say why you would do this.

Reply:
So I can also send multiple RST_STREAMs?

Reply:
It can, but they'd be indistinguishable from a retransmission